editor/svgutils.js

```diff
@@ -634,6 +634,13 @@ svgedit.utilities.cleanupElement = function(element) {
 		'ry':0
 	};
 
+	var nodeName = element.nodeName ? element.nodeName : '';
+	if (nodeName == 'ellipse') {
```

Comment:
(Nit) this should do the trick without creating a local string:
if (element.nodeName === 'ellipse') {